haystack/utils.py

```diff
@@ -29,6 +29,39 @@ def launch_es():
         time.sleep(15)
 
 
+def launch_milvus():
+    # Start a Milvus server
+    # You can start Elasticsearch on your local machine instance using Docker. If Docker is not readily available in
```

Comment:
Update comment here for milvus / remove the "executable part"

Reply:
What do you mean exactly by the "executable part"?